sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -1283,6 +1319,16 @@ export interface PathSetAccessControlOptions extends CommonOptions {
     owner?: string;
 }
 
+// @public
+export enum PathSetAccessControlRecursiveMode {
```

Comment:
could this be a string literal type? e.g.

```ts
export type PathSetAccessControlRecursiveMode = "modify" | "remove" | "set"
```

Reply:
For history reasons, storage datalake SDK uses eunm for generated types in generator options. We can set the option to false to generate string literal types in next major release, as it will break.

`PathSetAccessControlRecursiveMode` is an internal type used by private methods, let's me check if we can remove it from public types.